hack/verify-archives.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Can you add a blurb about what this script does? What's the expected input, what is the output, etc? Also include usage notes (how to invoke the script).

Reply:
This script extracts the golden image tarballs and compares their contents against the already-tracked golden folder (which is used by Bazel to generate deterministic images on every e2e test invocation). The purpose of this script is to verify that the tarballs residing in golden-archives matches the original source of truth (`golden`).

Reply:
ack